### PR TITLE
deprecating raspbian scripts, adding architecture docs, updating readme

### PR DESCRIPTION
**Pull request by xunholy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,99 +1,99 @@
...
+Once the Raspberry Pi's are running and all the prerequisites have been completed we're now ready to setup the Ansible inventory.
 
-At this point the Raspberry Pi should be running and on the same network as your machine so it can be accessed over ssh.
+Open the [inventory file](ansible/inventory) - each machine that will be joining the Kubernetes cluster must be defined as either a master or worker node. To leverage the highly available topology configuration you would ideally have 3 masters available as a minimum, otherwise 1 master node is fine, however, it won't be highly available.
 
-### Master Node(s)
+> Note: Ensure the `hostname` matches what the machine was given when flashed the SD card and `ansible_host` matches the IP allocated to the host on your prefered subnet.
 
-The master node configuration is copied from your machine to the Raspberry Pi with the following command:
+When the inventory has been configured with all the hosts that will be joining the Kubernetes cluster we can run the following command to verify SSH connectivity can be established.
```
Comment (omarst9) [inline README.md:28]: A link to a sample/current inventory would be helpful here

File: README.md [hunk 2]
```diff
@@ -1,99 +1,99 @@
...
-```bash
-RPI_HOSTNAME=k8s-master-01 KUBE_NODE_TYPE=master RPI_IP=192.168.1.101  make post-install
-```
+**Container Network Interface**
 
-The kube config and join token should now be in the ./output directory and will be used in configuring additional nodes.
+| Name | Supported |
+| ---- | --------- |
+| Weave | Yes |
+| Flannel | Yes |
```
Comment (omarst9) [inline README.md:63]: link to the file or a snippet showing how the different options can be configured

File: README.md (hunk 1 above)
Comment (xunholy) [reply README.md:28]: `[inventory file](ansible/inventory)`

File: README.md (hunk 2 above)
Comment (xunholy) [reply README.md:63]: currently link to vars file do you think i should go more specific and show it here as well? 

```
[vars.yml](ansible/vars.yml)
```

**Pull request merged**